### DISCUSSION TITLE
Account

### DISCUSSION THREAD

**Discussion by PTemp1e** [ctubio/Krypto-trading-bot#1164] [General]
Hi
my API disappeared from Bitfinex and I had to make a new one. Unfortunately, I no longer have an unlocked account. Would it be possible to unlock my new API. BTC address would be 3ERS8xqhQN1ouiSPzDm3feLyEZLn5NCqLe.
Many Thanks

**Comment by ctubio** [comment]
sure, 3ERS8x.. is now unlocked

**Reply by PTemp1e** [reply]
Thank you, as soon as I've found a setup that's profitable I'll reciprocate.
